packer/docs/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>Ed-Fi SEA Modernization Starter Kit</title>
+
+    <link rel="icon" href="https://edfidata.s3-us-west-2.amazonaws.com/Starter+Kits/images/cropped-logo-32x32.png" sizes="32x32" />
+    <link rel="icon" href="https://edfidata.s3-us-west-2.amazonaws.com/Starter+Kits/images/cropped-logo-192x192.png" sizes="192x192" />
+    <link rel="apple-touch-icon-precomposed" href="https://edfidata.s3-us-west-2.amazonaws.com/Starter+Kits/images/cropped-logo-180x180.png" />
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+    <style>
+        body {
+            font-family: 'Montserrat', sans-serif;
+            background-color: black;
+            color: white;
+        }
+
+        .p-l-10 {
+            padding-left: 10px;
+        }
+
+        .p-l-20 {
+            padding-left: 20px;
+        }
+
+        .inline {
+            display: inline;
+        }
+
+        strong {
+            font-weight: bold;
+        }
+
+        a {
+            color: #1bb7eb;
+            font-weight: bolder;
+        }
+
+        table, th, td {
+            border: 1px solid white;
+        }
+
+        table {
+            border-collapse: collapse
+        }
+
+        th, td {
+            padding: 5px;
+        }
+    </style>
+</head>
+
+<body>
+    <img src="https://edfidata.s3-us-west-2.amazonaws.com/Starter+Kits/images/EdFiDevServerBackground.png" style="width:80%; max-width:600px;" />
+    <div class="p-l-20">
+        <h1>This Development Server Contains the Ed-Fi ODS / API & the SEA Modernization Starter Kit</h1>
+        <p>
+            The SEA Modernization Starter Kit provides step-by-step instructions to technical and non-technical users on how to implement the Membership and Special Education Student Data Collection process.
+        </p>
+        <br />
+
+        <h2>To learn more on how to use "The SEA Modernization Starter Kit" please review the Ed-Fi Techdocs article: <a href="https://techdocs.ed-fi.org/display/SK/SEA+Modernization+Starter+Kit">"SEA Modernization Starter Kit"</a></h2>
+        <hr />
+
+        <h4>SharedInstance Assets</h4>
+        <ul>
+            <h3>Suite 3 v5.2.0</h3>
+            <li>
+                API - <a href="https://@@DOMAINNAME@@/WebApi" target="_blank">(Click here)</a>
+                <ul>
+                    <li>Key: dqMi66UoBfLC</li>
+                    <li>Secret: j65JETyk1WNmWKd9tbqMtPUt</li>
```

Comment:
this should be generated and/or loaded from a file not checked in

Reply:
@jamessmckay confirmed with Vinaya on removing the credentials from landing page. I have removed this now